### PR TITLE
correction of typo 'OrignalName' to 'OriginalName' in recording-function

### PR DESCRIPTION
**Pull request by ElsDeBie**
- correction of typo 'OrignalName' in the recording-function
- added possibility to chose the qualifier_type (site or management), comparable as rec_type in header-function and added differentiation in QualifierCode and QualifierDescription by adding Q1, Q2, Q3 in qualifier-function
- added possibility to get information of more than one suvey and added 'Observer' in the header-function

### CODE REVIEW

File: R/inboveg_header.R [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
 #' @description This function queries the INBOVEG database for header
-#' information (metadata for a vegetation-relevé) for one survey by the name
-#' of the survey and the recorder type. See the examples for how to get
-#' information for all surveys.
+#' information (metadata for a vegetation-relevé) for one or more surveys and
+#' the recorder type. See the examples for how to get information for all surveys.
 #'
-#' @param survey_name A character vector giving the name of the survey for which
-#' you want to extract header information. If missing, all surveys are returned.
+#' @param survey_name A character string or a character vector, depending on multiple parameter,
```
Comment (hansvancalster) [inline R/inboveg_header.R]:
```suggestion
#' @param survey_name A character string or a character vector  
```

File: R/inboveg_header.R [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+#' If missing, all surveys are returned.
 #' @param rec_type A character vector giving the name of record type for which
 #' you want to extract header information e.g. 'Classic', 'Classic-emmer',
 #' 'Classic-ketting', 'BioHab', 'ABS'. If missing, all recording types are returned.
 #' @param connection dbconnection with the database 'Cydonia'
 #' on the inbo-sql07-prd server
+#' @param multiple If TRUE, survey_name can take a character vector with
+#' multiple survey names that must match exactly. If FALSE (the default),
+#' survey_name must be a single character string (one survey name) that can
+#' include wildcarts to allow partial matches
```
Comment (hansvancalster) [inline R/inboveg_header.R]:
```suggestion
#' include wildcards to allow partial matches
```

**Review by hansvancalster — CHANGES_REQUESTED** [review]
Very good improvements! Can you address the requested changes?

**Comment by hansvancalster** [comment]
@ElsDeBie can you also change the letter "é" in `inboveg_recording()` to "\u00E9". Otherwise é will not be displayed correctly in the help file. To make the help files:
````
install.packages("roxygen2") #just make sure you have version 7.0.2
roxygen2::roxygenise()
````

After that is done, also commit the newly created inboveg_*.Rd files and push them to this branch.

**Comment by florisvdh** [comment]
@hansvancalster did you test the 'é' conversion? That trick did not work [here](https://github.com/inbo/inborutils/pull/66).

**Comment by hansvancalster** [comment]
Indeed, forgot that that didn't work. But it is circumvented now by ee1cdec

**Review by hansvancalster — APPROVED** [review]

**Pull request merged**